docs/api/api/delete_excluded_requests.xml

```diff
@@ -1,3 +1,3 @@
 <excluded_requests>
-  <number>33</number>
+  <request>33</request>
```

Comment:
this should be `<request id='33'/>` if you want to be consistent

Reply:
I only saw the initial version of [your comment](https://github.com/openSUSE/open-build-service/issues/8732#issuecomment-555484334). On it.

Reply:
Done.